src/framework/components/particlesystem/particlesystem_system.js

```diff
@@ -197,10 +197,35 @@ pc.extend(pc, function() {
             }, {
                 name: "spawnBounds",
                 displayName: "Spawn Bounds",
-                description: "Defines an AABB in which particles are allowed to spawn",
+                description: "Defines an AABB/radius in which particles are allowed to spawn",
```

Comment:
I would say, yes, these have to be different properties that we hide/show. I think 'spawnBounds' was OK before, but now, with multiple shapes, I think we should rename to: 'Emitter Shape' (emitterShape). As for the naming of the size properties.....that's harder. Let's take the radius. I think there are 4 options: 'Spawn Radius', 'Spawn Bounds', 'Emitter Radius' or 'Radius'. I think Emitter Radius is the most precise name. As for the Box type, is spawnBounds currently Extents or Half Extents? @vkalpias and @Maksims  - can you give your thoughts here?